src/contact/adapters/contactFormAdapter.ts

```diff
@@ -50,8 +52,59 @@ ${formData.get('context') || ''}`;
   return output;
 };
 
+export type Suggestion =
+  | 'update'
+  | 'PDB'
+  | 'buy'
+  | 'blast'
+  | 'align'
+  | 'id mapping'
+  | 'peptide search';
+const entryUpdateRegExp = /entry update request/i;
+const pdbRegExp = /(structure)|(pdb)/i;
+const buyRegExp = /(buy)|(purchas)|(ship)|(cost)|(quote)/i;
+const blastRegExp = /blast/i;
+const alignRegExp = /(clustal)|(align)/i;
+const idMappingRegExp = /(map)/i;
+const peptideSearchRegExp = /(peptide search)|(search peptide)/i;
+
+export const getSuggestion = throttle(
+  (subject: string, message: string): undefined | Suggestion => {
+    if (entryUpdateRegExp.test(subject)) {
+      return 'update';
+    }
+    if (pdbRegExp.test(message)) {
+      return 'PDB';
+    }
```

Comment:
Just doublechecking that you didn't want to check the message for subject and message in both of these?